### PR TITLE
Resolve code climate issues.

### PR DESCRIPTION
**Pull request by floor114**

### CODE REVIEW

**Comment by floor114** [comment]
Fix rubocop issues

**Review by sveredyuk — DISMISSED** [review]

**Review by sveredyuk — DISMISSED** [review]
GJ!

**Review by denys-medynskyi — DISMISSED** [review]
Awesome work!

File: app/assets/javascripts/app/application.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
 //= require lory
 //= require_tree .
 
-(function() {
+$(function() {
```
Comment (sveredyuk) [inline app/assets/javascripts/app/application.js]: remove, we not using jquery

File: app/assets/javascripts/app/change_img_color.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(function() {
+$(function() {
```
Comment (sveredyuk) [inline app/assets/javascripts/app/change_img_color.js]: remove, we not using jquery

File: app/assets/javascripts/app/dropdowns.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-(function(){
+$(function(){
```
Comment (sveredyuk) [inline app/assets/javascripts/app/dropdowns.js]: same

File: app/assets/javascripts/app/image_bending.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
-(function() {
+$(function() {
```
Comment (sveredyuk) [inline app/assets/javascripts/app/image_bending.js]: no jquery

File: app/assets/javascripts/app/slider.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-(function() {
+$(function() {
```
Comment (sveredyuk) [inline app/assets/javascripts/app/slider.js]: no jquery

**Review by sveredyuk — COMMENTED** [review]
Fix "no jquery" please

**Comment by floor114** [comment]
@sveredyuk Ok, Should I leave this issues not fixed? Or fix it something else?

**Comment by sveredyuk** [comment]
@floor114 Can we configure js-linter to skip `$` require?

**Comment by floor114** [comment]
@sveredyuk I think yes

**Comment by sveredyuk** [comment]
Lets try merge this

**Comment by floor114** [comment]
@sveredyuk not now, I fogot to remove dist: trusty

**Comment by floor114** [comment]
@sveredyuk done. Now we can merge it.

**Pull request merged**